src/header/navigation.rs

```diff
@@ -16,6 +16,28 @@ pub fn Header() -> impl IntoView {
             href: "/transactions".to_string(),
             text: "Transactions".to_string(),
             icon: NavIcon::Transactions,
+            sub_entries: Some(vec![
+                NavEntry {
+                    href: "/transactions".to_string(),
+                    text: "Transactions".to_string(),
+                    icon: NavIcon::Transactions,
+                    ..Default::default()
+                },
+                NavEntry {
+                    href: "/transactions/token-trxs".to_string(),
+                    text: "Token Trnxs".to_string(),
```

Comment:
I know you are trying to keep the label short, but I'm not a huge fan of `Trnxs` as an abbreviation for `Transaction`. Is it too long to fully spell out?

Reply:
It can be spelled out fully, yes. It does get long, visually.